fedora_distro_aliases/__init__.py

```diff
@@ -87,6 +95,16 @@ def get_distro_aliases(cache: Optional[SaveLoad] = None):
         "epel-all": epel,
     }
 
+    # Generate aliases for minor versions of EPEL
+    epel_branching = [x for x in epel if x.major_version >= 10]
+    key = operator.attrgetter("major_version")
+    for majorver, group in itertools.groupby(sorted(epel_branching, key=key), key=key):
+        aliases[f"epel-{majorver}-all"] = minors = list(group)
+        aliases[f"epel-{majorver}-branched"] = [x for x in minors if "." in x.branch]
```

Comment:
Apparently, at some point in the future for a short period of time there will be 3 minor releases, all in `active` state. So this has to be adjusted.

Reply:
Maybe I am misunderstanding again, but I imagined that in the future when were are on e.g. RHEL 10.7, this will be:

```
>>> [x.namever for x in aliases["epel-10-all"]]
["epel-10.0", "epel-10.1", "epel-10.2", "epel-10.3", "epel-10.4", "epel-10.5", "epel-10.6", "epel-10.7"],  # plus maybe 10.8
```

This is incorrect?
Feel free to ping me off-list and we can do a short call about this.

Reply:
Yes, that's incorrect.  At the time of RHEL 10.7 release, `epel-10-all` with include only `epel-10.6` (a.k.a. `epel-10-branched`) and `epel-10.7` (a.k.a. `epel-10`).
At any point, there are always only two active minor releases, the leading one and the trailing one (but, according to @carlwgeorge, at the point of branching a new minor release for a short while there will be 3 active minor releases in Bodhi, so one of them should be ignored - and I'm not actually sure which one).

Reply:
Thank you, that makes sense. I didn't realize that not all of them will be active at that point

> for a short while there will be 3 active minor releases in Bodhi, so one of them should be ignored - and I'm not actually sure which one).

No problem, we can IMHO address that once it happens.

Reply:
@FrostyX Thanks for merging this. Could you please do a new release as well?